src/bors/handlers/refresh.rs

```diff
@@ -3,24 +3,32 @@ use std::time::Duration;
 
 use anyhow::Context;
 use chrono::{DateTime, Utc};
+use futures::FutureExt;
+use futures::future::join_all;
 
 use crate::bors::Comment;
 use crate::bors::RepositoryState;
 use crate::bors::handlers::trybuild::cancel_build_workflows;
+use crate::bors::mergeable_queue::MergeableQueueSender;
 use crate::database::BuildStatus;
 use crate::{PgDbClient, TeamApiClient};
 
 pub async fn refresh_repository(
     repo: Arc<RepositoryState>,
     db: Arc<PgDbClient>,
     team_api_client: &TeamApiClient,
+    mergeable_queue_tx: MergeableQueueSender,
 ) -> anyhow::Result<()> {
     let repo = repo.as_ref();
-    if let (Ok(_), _, Ok(_)) = tokio::join!(
-        cancel_timed_out_builds(repo, db.as_ref()),
-        reload_permission(repo, team_api_client),
-        reload_config(repo)
-    ) {
+    let results = join_all([
+        cancel_timed_out_builds(repo, db.as_ref()).boxed(),
+        reload_permission(repo, team_api_client).boxed(),
+        reload_config(repo).boxed(),
+        reload_unknown_mergeable_prs(repo, db.as_ref(), mergeable_queue_tx).boxed(),
+    ])
+    .await;
+
+    if results.iter().all(|result| result.is_ok()) {
```

Comment:
We're swallowing errors here, we should log them instead. It was pre-existing though, so let's fix it in a separate PR.